PopoverMenu.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useRef, useState, useEffect } from 'react';
+// This is an SVG of three dots - it has gone through preprocessing and will render when used as <EllipsesMenuIcon />
+import EllipsesMenuIcon from './EllipsesMenuIcon.svg';
+
+interface PopoverMenuItem {
```

Comment:
Testing to see if i can leave a comment here

Reply:
got it! thank you for testing :)